blogposts/2021/show-us-your-calendar.md

```diff
@@ -0,0 +1,166 @@
+
+Loïc believes this so strongly that he’d refuse an offer from a company based on flexibility alone. I asked him to consider an inflexible dream job but he disagreed with the premise: “I doubt that it would be my dream job if it’s completely inflexible.”
+
+Flexibility isn’t just a benefit to be tallied up against other benefits–it says something about your company. ”In our industry,” Guychard argues, ”if you’re inflexible, you’re ignoring how we actually work and how software actually gets produced. Software is not something that gets produced on an inflexible schedule.”
+
+## Stephen Gutekanst, software engineer, takes “workations”
+
+[Stephen Gutekanst](https://about.sourcegraph.com/handbook/company/team#stephen-gutekanst), software engineer at Sourcegraph, gets bored during vacation. As much as he’d like to relax, his distance away from meetings creates the perfect headspace for inspiration. That unique headspace inspired him to make a change.
+
+Every once in a while, on an ad hoc, as-needed basis, Gutekanst marks a vacation in his calendar, turns off Slack and Zoom, and focuses entirely on coding–for as much as two weeks at a time.
+
+The workplace can get noisy. For Gutekanst, it comes in waves.
+
+Over the course of two months, he says, half of his time might go toward helping other people: “It’s a weird conundrum because it’s a completely useful way to spend my time. On the other hand, it’s completely unrelated to what my focus area actually is, which is working on a product feature or something like that.”
+
+![Slack message from Stephen regarding his workation](https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/Screen%20Shot%202021-09-13%20at%201.19.29%20PM.png)
+
+<p class="font-italic text-center">
+Stephen’s message to the team about his workation.
+</p>
+
+Part of the reason for that comes from working at a [high-growth startup](https://about.sourcegraph.com/blog/announcing-sourcegraphs-series-d-round/): “A lot of the time, we don’t have an owner for an area and I’m the person with the most context, so it’s useful for me to help out there.”
+
+It’s useful, he says, but also a dangerous temptation: “It’s really tempting, as an engineer, to say ‘Yeah, this is the main thing I’m working on but oh, this person needs help or someone has a question, let me jump in and help them.’ You can end up losing context a lot of the time.”
+
+Workations are a way to combat distractions and regain focus. To make them work, Stephen builds a clear consensus about what he’s doing and about what others expect him to do. “I know what I need to work on, I just need the time to go and do it. In those situations, detaching from everything else can be really powerful.”
+
+He takes a workation “whenever there’s a time where I’m like ‘Hey, I’d love to just sit down and write this code for a while.’” The result is a situation “where you can completely focus and get a lot done.”
```

Comment:
@slimsag The calendar post you contributed to is almost ready to go live. Would you mind taking a quick glance at your section and letting me know if it's accurate?

Reply:
Looks great to me! I left one minor comment inline.